android/app/build.gradle

```diff
@@ -187,7 +197,7 @@ android {
     }
 
     variantFilter { variant ->
-        if (variant.buildType.name == "devRelease" || variant.buildType.name == "prodDebug") {
+        if (variant.name == "devRelease" || variant.name == "prodDebug") {
```

Comment:
Even though its more verbose, `variant.buildType` seems to be more explicit in comparison to `variant.name` any reason on why this should be changed?

Reply:
`variant.buildType.name` is either `release` or `debug` so this condition was not correct. `variant.name` combines the flavors + build type, e.g. `devRelease`, where the flavor can either be `dev`, `staging` or `prod`.